backend/bittan/bittan/services/swish/swish_payment_request.py

```diff
@@ -0,0 +1,86 @@
+from enum import Enum
+
+from django.utils.log import logging
```

Comment:
Using different logging module, maybe only use regular logging and not Django. Or the other way around.

Reply:
9e7e1d5cbd652c4674ea7b42ecd94daa4d9ed38d